packages/nouns-bots/src/utils.ts

```diff
@@ -49,6 +59,26 @@ export function getAuctionEndingSoonTweetText() {
   return `This auction is ending soon! Bid now at https://nouns.wtf`;
 }
 
+export function formatNewGovernanceProposalText(proposal: Proposal) {
+  return `A new NounsDAO proposal (#${proposal.id}) has been created: ${extractProposalTitle(
+    proposal,
+  )}`;
+}
+
+export function formatUpdatedGovernanceProposalStatusText(proposal: Proposal) {
+  return `The NounsDAO proposal has changed to status: ${proposal.status.toLocaleLowerCase()}`;
```

Comment:
Nit on all instances: it would b helpful to include the proposal id and title "The Nouns DAO proposal #123 (Build skate park)..." except for the proposal created message.